.github/workflows/integration_test.yml

```diff
@@ -52,5 +52,5 @@ jobs:
 
       - name: Integration test
         run: |
-          poetry run coverage run -m unittest discover tests.integration
-          poetry run coverage report -m --fail-under=70
+          poetry run poe test_integration
+          poetry run coverage report --fail-under=70
```

Comment:
Do you not need the `-m`?

Reply:
-m is used for show_missing = true but I believe our pyproject.toml defaults to that already. If you check the CI tests in this PR run, the missing column still exists 

`[tool.coverage.report]
show_missing = true
skip_covered = true
skip_empty = true
precision = 2`